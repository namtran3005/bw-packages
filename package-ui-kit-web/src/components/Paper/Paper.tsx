import * as React from 'react';
import {
  Root,
  RootProps as PaperProps,
  RootContent,
  shadows,
} from './Paper.styled';

const Paper: React.FunctionComponent<PaperProps> = ({
  children,
  ...cleanProps
}) => (
  <Root {...cleanProps}>
    <RootContent>{children}</RootContent>
  </Root>
);

export { Paper, PaperProps, shadows };
