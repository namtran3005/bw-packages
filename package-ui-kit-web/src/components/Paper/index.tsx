export { Paper, PaperProps, shadows } from './Paper';
