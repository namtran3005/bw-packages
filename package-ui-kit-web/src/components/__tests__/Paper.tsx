import * as React from 'react';
import { shallow } from 'enzyme';
import { Paper } from '../Paper';

const defaultProps = {};

describe('Paper', () => {
  it('should render paper', () => {
    const wrapper = shallow(<Paper {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
