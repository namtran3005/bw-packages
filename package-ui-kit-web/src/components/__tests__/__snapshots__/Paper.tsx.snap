// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper should render paper 1`] = `
<Paper>
  <styled.div />
</Paper>
`;
